### PR TITLE
Add windows target support to kt

### LINKED ISSUE #189: Add windows-latest to CI test job
We have ubuntu and macOS but windows was failing for some hermit reason https://github.com/TBD54566975/web5-rs/actions/runs/9017114663

Creating a ticket to come back to such that other work is unblocked

### PR DESCRIPTION
**Pull request by KendallWeihe**
Running pipeline will have a `kotlin-acceptance-testing` job which runs the full unit test suite on a `windows-latest` os

Closes #189

### CODE REVIEW

File: crates/web5/Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ base64 = { workspace = true }
 byteorder = "1.5.0"
 chrono = { workspace = true }
 ed25519-dalek = { version = "2.1.1", features = ["rand_core"] }
-josekit = "0.8.6"
+josekit = { version = "0.8.6", features = ["vendored"] }
```
Comment (KendallWeihe) [inline crates/web5/Cargo.toml:15]: cc @diehuxx

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -60,19 +60,18 @@ jobs:
...
-        uses: cashapp/activate-hermit@v1
-        with:
-          cache: true
+      - name: Install Rust 1.74.0
+        run: rustup toolchain install 1.74.0
+      - name: Set Rust 1.74.0 as default
+        run: rustup default 1.74.0
       - name: Test
-        run: just test
+        run: cargo test --package web5
```
Comment (diehuxx) [inline .github/workflows/ci.yml:74]: y no just no hermit

File: crates/web5/Cargo.toml (hunk 1 above)
Comment (diehuxx) [reply crates/web5/Cargo.toml:15]: noice

**Review by diehuxx — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (KendallWeihe) [reply .github/workflows/ci.yml:74]: I don't think hermit has windows support, or at least the github action was running into issues on the windows os

**Pull request merged**